### PR TITLE
Build musllinux packages for multiple platforms

### LINKED ISSUE #269: Publish musllinux wheels for aarch64
Hi, would you please consider publishing musllinux wheels for aarch64 in addition to x86_64? I have a branch where I added this capability (https://github.com/scalyr/orjson/tree/multiarch-musllinux), but per your contribution guidelines I am creating an issue first.

Thanks!

### PR DESCRIPTION
**Pull request by dlobue**
Update the musllinux workflow to support building packages for multiple
platforms. Start with just aarch64 and x86_64 to start with.

Resolves #269

### CODE REVIEW

File: .github/workflows/musllinux.yaml [hunk 1]
```diff
@@ -9,22 +9,40 @@ jobs:
...
       with:
-        container: quay.io/pypa/musllinux_1_1_x86_64:latest
         rust-toolchain: nightly-2022-06-01
-        target: x86_64-unknown-linux-musl
+        target: ${{ matrix.platform.target }}
         manylinux: musllinux_1_1
-        args: --release --strip --out dist --no-sdist --cargo-extra-args="--features=unstable-simd" -i python${{ matrix.python-version }}
+        args: --release --strip --out dist --no-sdist --cargo-extra-args="--features=unstable-simd" -i python${{ matrix.python.version }}
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```
Comment (ijl) [inline .github/workflows/musllinux.yaml:38]: Could this step run only on x86_64?

Comment (dlobue) [reply .github/workflows/musllinux.yaml:38]: Sure!

File: .github/workflows/musllinux.yaml [hunk 2]
```diff
@@ -9,22 +9,40 @@ jobs:
...
+          { version: '3.10' },
+        ]
+        platform:
+          - target: aarch64-unknown-linux-musl
+            arch: aarch64
+          - target: x86_64-unknown-linux-musl
+            arch: x86_64
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
```
[the remark this answers is not in the document]
Comment (dlobue) [reply .github/workflows/musllinux.yaml:27]: The cross-compile container that maturin-action uses only has python 3.8 installed. Maturin-action supports building for python versions that are installed in the tool cache: https://github.com/messense/maturin-action/blob/main/src/index.ts#L556-L566

I actually tried building without this step originally, and I couldn't build for other python versions until I added it.

File: .github/workflows/musllinux.yaml (hunk 1 above)
Comment (dlobue) [reply .github/workflows/musllinux.yaml:39]: Done!

**Comment by dlobue** [comment]
Thank you!

**Pull request closed without merging**